rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -168,13 +167,15 @@ public Set<String> supportedCapabilities() {
 [[cluster_features]]
 === Require or skip cluster features
 
-In the past, Elasticsearch has been over-reliant on release version checks all across its code base.
-This has become a limiting factor when deploying Elasticsearch in a more continuous fashion.
+Cluster features indicate a particular top-level functionality and are used for _internal_ checks and coordination
```

Comment:
I would stress this is internal, especially now that we have capabilities. Maybe top-level -> code-level? 
Or "Cluster features indicate a particular _internal_ functionality used to add specific checks and coordination within the Elasticsearch cluster, to enable a specific behaviour once the code supporting it is available on all nodes" - or something like this